config/routes.rb

```diff
@@ -49,6 +49,10 @@
   get "video" => "welcome#video"
   get "vote-for-us" => "welcome#nycbigapps"
 
+  namespace :admin do
+    resources :buildings
```

Comment:
It's under admin (synonymous with team member in this case) because these are the actions related to managing the data about buildings. My assumption was that advocates wouldn't be the ones who would be entering the data about a building.

We will definitely have separate public routes/building API that present the data about buildings.